scripts/post-release.sh

```diff
@@ -2,5 +2,6 @@
 set -eu
 
 git checkout master && git pull
-./scripts/bump-version.sh '' $(date -d "$(echo $CRAFT_NEW_VERSION | sed -e 's/^\([0-9]\{2\}\)\.\([0-9]\{1,2\}\)\.[0-9]\+$/20\1-\2-1/') 1 month" +%y.%-m.0.dev0)
+# shellcheck disable=SC2001
```

Comment:
What is this disabled check?

Reply:
@BYK https://github.com/koalaman/shellcheck/wiki/SC2001